patches/llvm/0002-Add-managarm-target.patch

```diff
@@ -533,13 +536,18 @@ index 000000000..3639920fd
 +      getMultiarchTriple(getDriver(), getTriple(), getDriver().SysRoot);
 +  const GCCVersion &Version = GCCInstallation.getVersion();
 +
-+  // The primary search for libstdc++ supports multiarch variants.
-+  if (addLibStdCXXIncludePaths(LibDir.str() + "/../include",
-+                               "/c++/" + Version.Text, TripleStr,
-+                               GCCMultiarchTriple, TargetMultiarchTriple,
-+                               Multilib.includeSuffix(), DriverArgs, CC1Args))
++  // Try generic GCC detection first.
++  if (Generic_GCC::addGCCLibStdCxxIncludePaths(DriverArgs, CC1Args,
++                                               TripleStr))
 +    return;
 +
++  // The primary search for libstdc++ supports multiarch variants.
++  // if (addLibStdCXXIncludePaths(LibDir.str() + "/../include",
++  //                              "/c++/" + Version.Text, TripleStr,
++  //                              GCCMultiarchTriple, TargetMultiarchTriple,
++  //                              Multilib.includeSuffix(), DriverArgs, CC1Args))
++  //   return;
++
```

Comment:
Where does this come from?

Reply:
The Generic GCC thing is from the Linux toolchain, and I couldn't find a matching call for the other if statement anywhere. As it doesn't compile and my initial fix is what is done a bit further down in the patch already, I figured we could probably remove it.

Reply:
So the "primary search for libstdc++" comes from our old patch and has no equivalent in the Linux toolchain anymore? If yes, we can probably delete it.

Reply:
Seems like it. I'll delete it.